patroni/ctl.py

```diff
@@ -1035,3 +1046,36 @@ def show_config(obj, cluster_name):
     cluster = get_dcs(obj, cluster_name).get_cluster()
 
     click.echo(format_config_for_editing(cluster.config.data))
+
+
+@ctl.command('version', help='Output version of patronictl command or a running Patroni instance')
+@click.argument('cluster_name', required=False)
+@click.argument('member_names', nargs=-1)
+@click.pass_obj
+def version(obj, cluster_name, member_names):
+    click.echo("patronictl version {0}".format(__version__))
+
+    if not cluster_name:
+        return
+
+    click.echo("")
+    cluster = get_dcs(obj, cluster_name).get_cluster()
+    for m in cluster.members:
+        if m.api_url:
+            if not member_names or m.name in member_names:
+                try:
+                    response = request_patroni(m, 'get', None, auth_header(obj))
```

Comment:
Here you are trying to send `{'Authorization': 'Basic base64encoded'} ` as a payload of a GET request, what doesn't make any sense.
I guess you wanted to execute request `GET http://host:port/patroni`, because it will always return http code = 200
It should be done as `response = request_patroni(m, 'get', 'patroni')`.

The rest looks fine to me.

Reply:
Indeed. No idea why I was doing that. Will fix.